src/js/cookie-message.js

```diff
@@ -56,12 +56,19 @@ class CookieMessage {
 
 		this.options.theme = this.options.theme ? 'alternative' : null;
 
-		if (this.shouldShowCookieMessage()) {
-			this.createCookieMessage();
-			this.showCookieMessage();
-		} else {
-			this.removeCookieMessage();
-		}
+		window.addEventListener("pageshow", (event) => {
+			// If the FTConsentGDPR cookie is readable at this point we can omit this clause
+			if (event.persisted === true) {
```

Comment:
We should add a comment to explain what having a true value for `persisted` means.

Something like:
```suggestion
			// The persisted property being true indicates that the webpage was loaded from the browsers cache.
			if (event.persisted === true) {
```